src/shared/components/donate.tsx

```diff
@@ -81,43 +81,53 @@ export const DonateButtons = () => (
   </div>
 );
 
-const FundingGoal = () => (
-  <div className="flex flex-col flex-wrap mb-3 gap-4">
-    <div className="flex flex-row flex-wrap justify-between gap-4">
-      <div>
-        <span className="text-xl font-bold">
-          €{NUMBER_FORMAT.format(TOTAL_RECURRING_MONTHLY_EUR)}
-        </span>
-        <span className="text-gray-200 mr-3">
-          {i18n.t("per_month", { formattedCount: "" })}
-        </span>
-      </div>
-      <div
-        className="text-xl font-bold tooltip underline decoration-dotted"
-        data-tip={i18n.t("based_on_salary", {
-          formattedCount: NUMBER_FORMAT.format(MEDIAN_DEV_SALARY),
-        })}
-      >
-        {i18n.t("devs_funded", {
-          formattedCount1: FUNDED_DEVS.toFixed(1),
-          formattedCount2: FUNDED_DEV_GOAL,
-        })}
-      </div>
-    </div>
-    <progress
-      className="progress progress-primary w-auto"
-      value={FUNDED_DEVS}
-      max={FUNDED_DEV_GOAL}
-    ></progress>
-    <div className="flex flex-row flex-wrap justify-between gap-4">
-      <div className="text-sm text-gray-300">
-        {i18n.t("supporters", {
-          formattedCount: NUMBER_FORMAT.format(TOTAL_SUPPORTERS),
-        })}
+const FundingGoal = () => {
+  var progress_classes = "h-5 progress w-auto mb-6 ";
+  var text_classes = "flex flex-row flex-wrap justify-between gap-4 ";
+  if (TOTAL_RECURRING_MONTHLY_EUR < 4000) {
+    progress_classes += "progress-error";
+    text_classes += "text-error/75";
+  } else if (TOTAL_RECURRING_MONTHLY_EUR < 7000) {
+    progress_classes += "progress-warning";
+    text_classes += "text-warning";
+  } else {
+    progress_classes += "progress-success";
+    text_classes += "text-success";
+  }
```

Comment:
Might not work super-well here, but check out the https://github.com/JedWatson/classnames library sometime. We use it a ton in lemmy-ui. its really useful whenever you conditionally need to add classes.

Reply:
Seems unnecessary in this case.